frontend/src/styles/finsight-news-list.css

```
@@ -0,0 +1,391 @@
+
+.extra .content_area .list_top .thumb_type li:first-child{display:none}
+.extra .content_area .list_top .thumb_type li{margin-right:24px}
+.extra .content_area .list_top .thumb_type li:nth-child(4),
+.extra .content_area .list_top .thumb_type li:nth-child(6){margin-right:0}
+
+@media screen and (max-width: 320px){
+    .list_radio .time{width:28%}
+    .list_radio .r_name{width:42%}
+    .list_radio .r_vod{width:30%}
+}
+@media screen and (max-width: 360px){}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)